Uslov.Operator.Task4/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Uslov.Operator.Task4
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите x ");
+            var x = double.Parse(Console.ReadLine());
+            if(x<-3)
+            {
+                Console.WriteLine("f(x)=0");
```

Comment:
Вывод лучше смотрится, если знак равенства отделяется пробелами
```suggestion
                Console.WriteLine("f(x) = 0");
```